lib/sensors/src/AdafruitSTEMMA.cpp

```diff
@@ -1,12 +1,15 @@
 #include "AdafruitSTEMMA.h"
 
+#include <mutex>
```

Comment:
this driver is a mess. (a read can take up to 10 seconds lol). ill create a task to clean this up, but idk who'll do this